hydra/test_utils/launcher_common_tests.py

```diff
@@ -388,13 +424,16 @@ def test_custom_task_name(
         task_config = OmegaConf.create(task_config or {})  # type: ignore
         cfg = OmegaConf.merge(task_launcher_cfg, task_config)
         assert isinstance(cfg, DictConfig)
+        output_tmp_dir, overrides = self.filter_tmp_dir(overrides)
         integration_test(
             tmpdir=tmpdir,
             task_config=cfg,
             overrides=overrides,
             prints="HydraConfig.get().job.name",
             expected_outputs=expected_name,
             filename=filename,
+            output_tmp_dir=output_tmp_dir,
+            post_run_callback=self.post_run_callback(),
```

Comment:
this is calling the function, you probably want to pass it instead.

Reply:
```self.post_run_callback()``` returns another function here, so calling it here

Reply:
Please do.

Reply:
fixed this.